hypervisor/src/kvm/mod.rs

```diff
@@ -720,10 +720,7 @@ impl vm::Vm for KvmVm {
             .map_err(|e| vm::HypervisorVmError::SetGsiRouting(e.into()))
     }
 
-    ///
-    /// Creates a memory region structure that can be used with {create/remove}_user_memory_region
-    ///
-    fn make_user_memory_region(
+    unsafe fn make_user_memory_region(
         &self,
         slot: u32,
         guest_phys_addr: u64,
```

Comment:
Would it be better to use a `GuestAddress` here?

Reply:
It would be, but it would also be better to use `GuestAddress` in many other places.  That's orthogonal to the purpose of this PR, which is fixing unsound use of _host_ addresses.